src/components/flyout/Overlay.js

```diff
@@ -35,7 +36,7 @@ class Overlay extends React.Component<Props> {
     render() {
         const { children, className, ...rest } = this.props;
         const overlayProps = omit(rest, ['onClose']);
-        overlayProps.className = className;
+        overlayProps.className = classNames('overlay-content', className);
```

Comment:
```suggestion
        overlayProps.className = classNames('bdl-Overlay', className);
```

Reply:
@tjuanitas thoughts on adding this...at the risk of modifying a component to have inconsistent naming styling?

Reply:
yeah that change seems appropriate to me

any particular reason the class name was moved from `Flyout.js` to `Overlay.js`?

Reply:
I decided to move it there because making that change in Flyout seemed more complicated. In there, we duplicate the Overlay child and override the props. Making an update to Overlay's classname will now require me too grab the original className so I can make sure that it is propagated properly

```
const overlayProps = {
            className: classNames('flyout-overlay-content', overlayContent.className), // overlayContent.className doesn't work btw, just using it to prove my point
...
}
```